### ISSUE TITLE
'More Information' Icons

### ISSUE THREAD

**Issue by B5r1oJ0A9G** [william-gr/freenas-bacula#13]
Just noticed the 'More Information' icons in other setting windows.
I think the windows would look o lot more clearly if the comments were replaced by such icons.

**Comment by william-gr** [comment]
I agree but this is an feature of FreeNAS GUI that is really hard to achieve in plugins, too much trouble for so little, I may implement this someday but is _really_ low priority for me...
